tests/test_ena_handler.py

```diff
@@ -165,7 +169,7 @@ def test_get_study_assemblies_should_have_all_fields(self):
         ena = ena_handler.EnaApiHandler()
         assemblies = ena.get_study_assemblies('ERP112609')
```

Comment:
Just one comment that this accession seems to have only two suppressed assemblies. Another study might be better to use?

Reply:
Ah, good catch. I need to review the failling tests.. I'll change that.